modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -151,8 +151,8 @@ public static class Sql {
         /** Table version not found. */
         public static final int TABLE_VER_NOT_FOUND_ERR = SQL_ERR_GROUP.registerErrorCode(18);
 
-        /** Invalid table option. */
```

Comment:
"Create\Drop index" are also DDL operations, but
the error code related to "create table syntax only, don't it?

Reply:
> the error code related to "create table syntax only
Yes, at the moment, but I think this error code may relate to any invalid DDL statement option in future, why not?
But if you suggest making a separate error code for invalid `CREATE ZONE` options, I can add it.

Reply:
Thanks, the code has been replaced with `QUERY_VALIDATION_ERR`.  
`TABLE_OPTION_ERR` removed.